cmd/utils/nodecmd/util.go

```diff
@@ -0,0 +1,220 @@
+package nodecmd
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/klaytn/klaytn/accounts/keystore"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/cmd/utils"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/common/hexutil"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	"github.com/klaytn/klaytn/crypto"
+	"github.com/klaytn/klaytn/crypto/sha3"
+	"github.com/klaytn/klaytn/governance"
+	"github.com/klaytn/klaytn/params"
+	"github.com/klaytn/klaytn/rlp"
+	"gopkg.in/urfave/cli.v1"
+)
+
+const pgmInput = `
+<extra> <header file (json format)>
+<vote>  <bytes>
+<gov>   <bytes>
+<key>   <keystore path> <password>
+`
+
+var ErrInvalidParam = errors.New("Invalid length of parameter")
+
+var UtilCommand = cli.Command{
+	Action:    utils.MigrateFlags(parse),
+	Name:      "util",
+	Usage:     "offline utility " + pgmInput,
+	ArgsUsage: " ",
+	Category:  "MISCELLANEOUS COMMANDS",
+}
+
+func hex2Bytes(s string) []byte {
```

Comment:
I don't think this function is really needed. Why don't we just use hexutil.Decode directly?